Documentation/eliminating-pre-builts.md

```diff
@@ -0,0 +1,88 @@
+# Eliminating pre-builts in .NET repositories
+
+This is a detailed guide on how to eliminate pre-builts in a source-buildable repository.
+It is primarily intended for developers contributing to the `dotnet` organization.
+
+## Table of content
+  - [What is a Prebuilt](#what-is-a-prebuilt)
+  - [Elimitating pre-builts](#eliminating-pre-builts)
+  - [FAQ](#faq)
+  - [Contacts](#contacts)
+
+# What is a Prebuilt
+
+_Source-build_ is a process of building a given product on a single machine from source with no internet access.
+
+By definition, _pre-builts_ are dependencies that a repo has on binary files that are not built from source, where _build from source_ points to any package produced during the _current source-build_ or _already present_ in the distro at the time of building (such as `cmake`).
```

Comment:
> where _build from source_ points to any package produced during the _current source-build_ or _already present_ in the distro at the time of building (such as `cmake`)

This isn't real clear to me.  What about saying something like, _the exception to this are dependencies that are picked up from the host distro such as `cmake_`?

Reply:
Agree, changed; don't know if this is enough or the reason for why dependencies present on the host distro are allowed should also be added